keanu-project/src/main/java/io/improbable/keanu/vertices/Observable.java

```diff
@@ -14,17 +14,11 @@
 
     boolean isObserved();
 
-    static <T> Observable<T> observableTypeFor(Class<? extends Vertex> v) {
-
+    static boolean isObservable(Class<? extends Vertex> v) {
```

Comment:
Feels like `isObservable` should live on `Vertex` rather than on `Observable`

Reply:
done